tululbot/config.py

```diff
@@ -11,3 +11,9 @@
 except KeyError:
     warnings.warn('TULULBOT_DEVEL_CHAT_ID is not set; errors will pass silently')
     TULULBOT_DEVEL_CHAT_ID = ''
+
+try:
+    HOTLINE_MESSAGE_ID = environ['HOTLINE_MESSAGE_ID']
+except KeyError:
+    warnings.warn('HOTLINE_MESSAGE_ID is not set; errors will pass silently')
```

Comment:
I think the warning message is not right. Setting `HOTLINE_MESSAGE_ID` has nothing to do with errors, right?